microsoft-edge/develop-web-microsoft-edge.md

```diff
@@ -6,7 +6,7 @@ ms.author: msedgedevrel
 ms.topic: conceptual
```

Comment:
CapB wrote on main page of PR:

We should also think about renaming the "Web Platform" card on the landing page: https://learn.microsoft.com/en-us/microsoft-edge/developer/

Reply:
pushed fix